specs/MultiProfile.md

```diff
@@ -379,6 +423,8 @@ namespace Microsoft.Web.WebView2.Core
         String ProfilePath { get; };
 
         CoreWebView2CookieManager CookieManager { get; };
+        
+        public void Erase();
```

Comment:
Erase won't be in the same interface as CookieManager. If you run build-apiwriter.ps1 and look at the generated Microsoft.Web.WebView2.Core.IDL you should see Erase is in its own interface_name block. Please copy that here

Reply:
I have changed it, do I need to change "CookieManager" by the way?